application/.gitignore

```diff
@@ -1,9 +1,114 @@
+# Built application files
+*.apk
+*.aar
+*.ap_
+*.aab
+/*/build/
+
+# Crashlytics configuations
+com_crashlytics_export_strings.xml
+
+# Signing files
+.signing/
+
+# Files for the ART/Dalvik VM
+*.dex
+
+# Java class files
+*.class
+
+# Generated files
+bin/
+gen/
+out/
+#  Uncomment the following line in case you need and you don't have the release build type files in your app
+# release/
+
+# Gradle files
+.gradle/
+build/
```

Comment:
I need to be able to modify this to connect with fb so it should not be ignored when I push

Reply:
.gradle/ 
or 
build/
?